features/core-download.feature

```diff
@@ -95,3 +95,87 @@ Feature: Download WordPress
       """
       File removed: wp-content
       """
+
+  Scenario: Installing trunk
```

Comment:
I'd recommend erring if a non-en_US locale is provided. If someone is providing a locale, then they should receive an error that the nightly isn't provided in the locale, not have it unexpectedly work and download the wrong file.